content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -299,18 +299,18 @@ Implicit m-n relations:
 If you obtain your data model from [introspection](/concepts/components/introspection), you can still use implicit many-to-many relations by following Prisma's [conventions for relation tables](#conventions-for-relation-tables-in-implicit-m-n-relations).
 The following example assumes you want to create a relation table to get an implicit many-to-many relation for two models called `Post` and `Category`.
 
-##### Table name
+##### Table names
 
-The name of a relation table for an implicit-many-to-many relation is constructed from the following parts:
+If you want a relation table to be picked up by introspection as an implicit many-to-many relation, the name must follow this exact structure:
 
 - It must start with an underscore `_`
-- Then follows the alphabetically lower table name (in this case `Category`)
-- After which `To` connects to the rest of the name
-- Which is the alphavetically higher table name (in this case `Post`)
+- Then the name of the first table in alphabetical order (in this case `Category`)
+- Then `To`
+- Then the name of the second table in alphabetical order (in this case `Post`)
 
-In the example the correct table name is `_CategoryToPost`.
+In the example, the correct table name is `_CategoryToPost`.
 
-(When creating an implicit m-n relation via Prisma itself, you can also [configure the name of the relation table to be different](#configuring-the-name-of-the-relation-table-in-implicit-many-to-many-relations) to be for example `_MyRelationTable`)
+When creating an implicit many-to-many relation yourself in the Prisma schema file, you can [configure the relation name](#configuring-the-name-of-the-relation-table-in-implicit-many-to-many-relations) to have a different name, for example `_MyRelationTable`.
```

Comment:
You configure the relation name, but that changes the name that the table gets when being migrated. So it is kind of indirect but with exactly that result.

(Do we also add the underscore to these custom implicit m:n table named @do4gr?)

Reply:
You would give the name as `MyRelationTable` and we would then under the hood add the underscore to create a table named `_MyRelationTable`

Reply:
Thanks, clarified this.